src/SFA.DAS.PSRService.Web/Controllers/HomeController.cs

```diff
@@ -96,13 +96,16 @@ public IActionResult Error()
             return View(new ErrorViewModel {RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier});
         }
 
-        public async Task<IActionResult> Logout()
+        public async Task Logout()
```

Comment:
Doesn't this method still need to redirect to the employer users signout page?

Reply:
We were following the das-recruit approach and Lee's comment (from the Jira) : -

> Note: There is no explicit redirect. This is handled by the middleware.